tools/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+#This is required fo bypassing the tzdata installation
+ENV TZ=Europe/Bucharest
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
Any reason you did all these RUN's as separate lines? its better to have them all on one line if possible purely from a docker image size 

And you might want to remove the apt-repo list once youve updated

Reply:
Makes sense, I was not aware about this image size thing.  Now I've tried to group them a bit, so that the code still remains readable.